python-package/pyproject.toml

```diff
@@ -85,3 +85,9 @@ class-attribute-naming-style = "snake_case"
 
 # Allow single-letter variables
 variable-rgx = "[a-zA-Z_][a-z0-9_]{0,30}$"
+
+[tool.pydistcheck]
+inspect = true
+ignore = ["compiled-objects-have-debug-symbols"]
```

Comment:
Thanks for trying this out! If other checks in `pydistcheck` are not important to you or cause issues, you can add a section like this to opt in to only the ones you care about:

```toml
select = [
  "distro-too-large-compressed"
]
```

I modeled this off of the way `ignore` and `select` work in `ruff` ... if you populate `select`, it turns the behavior into "only run the checks listed in `select`, and skip all others".

That's what I did in PRs for https://github.com/rapidsai/build-planning/issues/110, to keep it tightly focused on that one purpose of checking file sizes.

If you want to keep all the other checks (https://pydistcheck.readthedocs.io/en/latest/check-reference.html) then great, I hope they're helpful, but just mentioning this so you know it's possible to limit the scope of what this tool is checking.